packages/super-components/components/msc-billing-tile/src/msc-billing-tile/msc-billing-contact.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+  Component,
+  h,
+  Prop,
+  State,
+  Host,
+  Element,
+  Fragment,
+} from '@stencil/core';
+import {
+  OdsThemeColorIntent,
+  OdsThemeTypographyLevel,
+  OdsThemeTypographySize,
+} from '@ovhcloud/ods-theming';
+import {
+  OdsHTMLAnchorElementTarget,
+  OdsIconName,
+  OdsIconSize,
+} from '@ovhcloud/ods-core';
+import { HTMLStencilElement } from '@stencil/core/internal';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import {
+  ContactInfos,
+  DomainProperties,
+  ServiceInfos,
+  Translations,
+} from './msc-billing.types';
+import {
+  getChangeOwnerUrl,
+  getContactManagementUrl,
+  getUpdateOwnerUrl,
+} from './urls';
+
+export interface IMscBillingContact {
+  serviceName: string;
+  serviceType: string;
+  serviceInfos: ServiceInfos;
+  changeOwnerDataTracking?: string;
+  updateOwnerDataTracking?: string;
+  subscriptionManagementDataTracking?: string;
+  localeStrings: Translations;
+}
+
+@Component({
+  tag: 'msc-billing-contact',
+  styleUrl: 'msc-billing-contact.scss',
+  shadow: true,
+})
+export class MscBillingContact implements IMscBillingContact {
+  @Element() host!: HTMLStencilElement;
+
+  @Prop() serviceName: string;
+
+  @Prop() serviceType: string;
+
+  @Prop() serviceInfos: ServiceInfos;
+
+  @Prop() changeOwnerDataTracking?: string;
+
+  @Prop() updateOwnerDataTracking?: string;
+
+  @Prop() subscriptionManagementDataTracking?: string;
+
+  @Prop() localeStrings: Translations;
+
+  @State() domainProperties?: DomainProperties;
+
+  @State() ownerInfos?: ContactInfos;
+
+  async componentWillLoad() {
+    if (this.isDomainServiceType()) {
+      this.domainProperties = await apiClient.v6
+        .get(`/domain/${this.serviceName}`)
+        .then((response) => response.data);
+
+      this.ownerInfos = await apiClient.v6
+        .get(`/domain/contact/${this.domainProperties?.whoisOwner}`)
+        .then((response) => response.data);
+    }
+  }
+
+  private isDomainServiceType() {
+    return this.serviceType === 'DOMAIN';
+  }
+
+  private isOwnerChangeableServiceType() {
+    return [
+      'DOMAIN',
+      'HOSTING_WEB',
+      'HOSTING_PRIVATEDATABASE',
+      'DEDICATED_SERVER',
+      'DEDICATED_HOUSING',
+      'VPS',
+      'EMAIL_DOMAIN',
+      'VRACK',
+      'DEDICATEDCLOUD', // VMWare
+    ].includes(this.serviceType);
```

Comment:
**nitpick:** Super-component cannot maintain the list of products like this. It can accept an input whether the owner can be changed or not.

Reply:
Why not ?
Why don't we include this domain rule in the component instead of delegating it to the user of the component ?
That will make it harder to maintain domain rules related to the subscription if they are spread among all apps